Tests/SwiftLintFrameworkTests/VerticalWhitespaceRuleTests.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  VerticalWhitespaceRuleTests.swift
+//  SwiftLint
+//
+//  Created by Aaron McTavish on 01/05/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+@testable import SwiftLintFramework
+import XCTest
+
+class VerticalWhitespaceRuleTests: XCTestCase {
+
+    func testVerticalWhitespaceWithDefaultConfiguration() {
```

Comment:
you should remove `testVerticalWhitespace` from `RulesTests` since it has moved to here